tests/pipelines/stable_diffusion/test_stable_diffusion_latent_upscaler.py

```diff
@@ -0,0 +1,218 @@
+# coding=utf-8
```

Comment:
oh actually I think the tests for latent upscaler is here https://github.com/huggingface/diffusers/blob/main/tests/pipelines/stable_diffusion_2/test_stable_diffusion_latent_upscale.py

maybe we add new tests there? very sorry for all the additional work to create a new test from scratch